### PR TITLE
Use rancid 3.x's field separator

### LINKED ISSUE #10: Is this working with rancid 3.X?
I had recently to re-install one of our rancid servers and I noticed that all routers were reported as down.
After a bit of digging I am still unclear where the problem lies.
I noticed that /usr/libexec/rancid/control_rancid changes the logic/format of temporary files, but the original router.db has still the same format.

Did anyone successfully use this module with rancid 3.X?

### PR DESCRIPTION
**Pull request by nhemingway**
Closes #10

### CODE REVIEW

File: spec/defines/router_db_spec.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+describe 'rancid::router_db' do
+  let(:title) { 'group1' }
+
+  let(:facts) { { :osfamily => 'Debian' } }
+  let(:pre_condition) {
+    'class rancid {
```
Comment (ghoneycutt) [inline spec/defines/router_db_spec.rb:7]: Should not need a pre-condition to set rancid variables.  Really should not need the `include rancid` as the defined type should have that in it if it is necessary.

File: spec/defines/router_db_spec.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        $homedir_real = "/tmp/rancid"
+        $user_real = "rancid_user"
+        $group_real = "rancid_group"
+      }
+      include rancid
+      '
+  }
+
+  context 'Use correct FS' do
```
Comment (ghoneycutt) [inline spec/defines/router_db_spec.rb:16]: FS?

**Review by ghoneycutt — CHANGES_REQUESTED** [review]
This should also have a change to README to indicate it only works with RANCID >= 3 unless you want to add a custom fact and make the separator token change dynamically.

File: spec/defines/router_db_spec.rb (hunk 1 above)
Comment (nhemingway) [reply spec/defines/router_db_spec.rb:7]: Except that in the real world, `router_db` is only intended to be invoked _from_ class rancid, and hence it is implicitly included.  Here, in unit tests, that's not the case, so I'm taking steps to setup the environment that `router_db` expects.

Comment (ghoneycutt) [reply spec/defines/router_db_spec.rb:7]: Just add `include rancid` to the defined type. Its absence is not by design.

Comment (ghoneycutt) [reply spec/defines/router_db_spec.rb:7]: Now that you have `include rancid` in the defined type, you should not need a `pre_condition` at all.

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

**Review by ghoneycutt — APPROVED** [review]

**Comment by ghoneycutt** [comment]
Well done!

**Pull request merged**